mappings/net/minecraft/client/render/entity/animation/WardenAnimations.mapping

```diff
@@ -4,3 +4,4 @@ CLASS net/minecraft/class_7278 net/minecraft/client/render/entity/animation/Ward
 	FIELD field_38304 ROARING Lnet/minecraft/class_7184;
 	FIELD field_38305 SNIFFING Lnet/minecraft/class_7184;
 	FIELD field_38306 ATTACKING Lnet/minecraft/class_7184;
+	FIELD field_38879 SONIC_BOOMING Lnet/minecraft/class_7184;
```

Comment:
Could this field name use an actual verb? For example (since the animation starts when the `minecraft:entity.warden.sonic_charge` sound event is played):

```suggestion
	FIELD field_38879 CHARGING_SONIC_BOOM Lnet/minecraft/class_7184;
```

If this field is renamed, make sure to rename `field_38859` accordingly.